src/v1/components/surfaces/FixedPanel/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { HTMLAttributes } from "react";
+import MUIStack from "@mui/material/Stack";
+import MUIBox from "@mui/material/Box";
+import { useTheme } from "@mui/material/styles";
+
+import { DownArrowIcon, Text } from "../../data-display";
+import { IconButton } from "../../inputs";
+import { useFloatingPanelContext } from "../FloatingPanel/useFloatingPanelContext";
+
+interface PanelProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The id target reference for the toggle button.
+   */
+  targetId: string;
+  /**
+   * 	Panel title.
+   */
+  title: string;
+  /**
+   * Total number of items
+   */
+  count?: number;
+  /**
+   * Component content
+   */
+  children?: React.ReactNode;
+  /**
+   *
+   */
+  dragHandle?: React.ReactNode;
+  /**
+   * Specifies a selector to be used as the handle that initiates drag. Example:
+   * drag-handle.
+   */
+  dragHandleClassName?: string;
+  /**
+   * The icon to be displayed before the title
+   */
+  icon?: React.ReactNode;
```

Comment:
```suggestion
  iconBeforeTitle?: React.ReactNode;
```

Reply:
I think it pays to be very literal with names of vars that are presentational/not strictly semantic